src/UI/PlayerSetupInfo.java

```diff
@@ -80,10 +80,6 @@ public Commander makeCommander(GameScenario.GameRules rules)
     co.faction = new Faction(availableFactions[currentFaction].name, availableFactions[currentFaction].basis);
     co.faction.flip = flipUnits;
 
-    co.team = currentTeam;
-
-    co.setAIController(availableAis[currentAi].create(co));
```

Comment:
Why move these assignments up to `GameBuilder`? I guess now you need the actual `Army` to pass to the AI, but the team doesn't seem to need that treatment.

Reply:
It seemed like a good idea to co-locate all the relevant initialization.